docs/reference/deploy/helm-config-reference.rst

```diff
@@ -231,10 +231,12 @@
    any HTTP traffic. The default timeout is 300 (5 minutes).
 
 -  ``initialUserPassword``: Specifies a string containing the default password for the admin and
-   determined user accounts.
+   determined user accounts in a brand new deployment. (*Required* when the database is first
```

Comment:
This makes it sound like it's required when the database is first bootstrapped, but it's required always and only effective then?

I guess mostly I'm pointing out that this is slightly weird to require when it will have no affect. I'm not sure how to fix that easily so we should at least make it clear to the user.

Reply:
Good point, that was not well written. Making another attempt; I suppose it doesn't hurt for these to be a little verbose. It's kind of unfortunate that we have to do it this way.